orchestration/hca_orchestration/repositories/base_repositories.py

```diff
@@ -36,14 +36,11 @@ def copy_project_to_new_dataset_job() -> PipelineDefinition:
         })
 
 
-def base_jobs() -> list[Union[PipelineDefinition, SensorDefinition]]:
+def base_jobs(mode: str) -> list[Union[PipelineDefinition, SensorDefinition]]:
     defs = [
         cut_snapshot,
         load_hca,
-        validate_egress,
-        build_post_import_sensor(os.environ.get("ENV", "test")),
```

Comment:
build_post_import_sensor is not used anymore (it was triggered when our old argo stuff ran).
copy_project_to_new_dataset_job only makes sense in a dev context.